docs/spec/channel/version_070.md

```diff
@@ -0,0 +1,140 @@
+  name: kafka-channelable-manipulator
+  labels:
+    duck.knative.dev/channelable: "true"
+# Do not use this role directly. These rules will be added to the "channelable-manipulator" role.
+rules:
+  - apiGroups:
+      - messaging.knative.dev
+    resources:
+      - kafkachannels
+      - kafkachannels/status
+    verbs:
+      - create
+      - get
+      - list
+      - watch
+      - update
+      - patch
+```
+
+Each channel MUST have the `duck.knative.dev/channelable: "true"` label on its `channelable-manipulator` CR file.
+
+#### CustomResourceDefinition per Channel
+
+For each channel implementation a `CustomResourceDefinition` is created, like:
+
+```
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+ name: kafkachannels.messaging.knative.dev
+ labels:
+    knative.dev/crd-install: "true"
+    messaging.knative.dev/subscribable: "true"
+spec:
+  group: messaging.knative.dev
+  version: v1alpha1
+  names:
+    kind: KafkaChannel
+    plural: kafkachannels
+    singular: kafkachannel
+    categories:
+    - all
+    - knative
+    - messaging
+    - channel
+    shortNames:
+    - kc
+  scope: Namespaced
+...
+```
+
+Each channel is _namespaced_ and MUST have the following:
+
+* label of `messaging.knative.dev/subscribable: "true"`
+* a `channel` category
+
+#### Broker and Triggers
+
+TODO
+
+### Data Plane
+
+The data plane describes the input and output flow of a _Channel_. All Channels exclusively communicate using CloudEvents.
+
+#### Input
+
+Every Channel must expose either an HTTP or HTTPS endpoint. It MAY expose both. The endpoint(s) MUST conform to [HTTP Transport Binding for CloudEvents - Version 0.2](https://github.com/cloudevents/spec/blob/v0.2/http-transport-binding.md). It MUST support both Binary Content mode and Structured Content mode. The HTTP(S) endpoint MAY be on any port, not just the standard 80 and 443. Channels MAY expose other, non-HTTP endpoints in addition to HTTP at their discretion (e.g. expose a gRPC endpoint to accept events).
+
+##### Generic
+
+If a Channel receives an event queueing request and is unable to parse a valid CloudEvent, then it MUST reject the request.
+
+Every event queueing request to the Channel will come with a bearer token, likely a JWT. The bearer token MUST be validated before any other work is done on the request. The specifics of how and what to validate will be identical to Broker ingress verification, which is being [defined](https://github.com/knative/eventing/issues/705#issuecomment-496722527).
+
+The Channel MUST pass through all tracing information as CloudEvents attributes. In particular, it MUST translate any incoming OpenTracing or B3 headers to the [Distributed Tracing Extension](https://github.com/cloudevents/spec/blob/v0.2/extensions/distributed-tracing.md). The Channel SHOULD sample and write traces to the location specified in [`config-tracing`](https://github.com/cloudevents/spec/blob/v0.2/extensions/distributed-tracing.md).
```

Comment:
config-tracing link is incorrect, although I don't know the correct one for now.